lib/pages/dashboard_page.dart

```diff
@@ -163,7 +163,7 @@ class _DashboardPageState extends State<DashboardPage> {
           constraints: constraints,
           menuWidth: menuWidth,
         ),
-      if (listItemTitle == 1) Container(),
+      if (listItemTitle == 1) PurchasePage(),
       if (listItemTitle == 2) Container(),
       if (listItemTitle == 3) Container(),
```

Comment:
@ahamzatariq , This should be a string that resides in a constants folder. 

`listItemTitle == constants.Purchase` , Something like this.

Reply:
Using Enums instead of int.
Implemented here in a commit in [PR#1](https://github.com/ahamzatariq/school_cafeteria_inventory_management_system/pull/1/commits/d7f1dba561d875304d18a263ed35b5de7d11fac0)